styles/styles.css

```diff
@@ -364,6 +367,18 @@ main img {
   vertical-align: middle;
 }
 
+[class*=" vlt-icon-"], [class^="vlt-icon-"] {
+  font-family: var(--icon-font-family);
+  font-style: normal;
+  font-variant: normal;
+  font-weight: 400;
+  line-height: inherit;
+  speak: none;
+  text-transform: none;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
```

Comment:
are these prefixed styles really needed?

Reply:
No they are not, tried to pull them out as I was collecting styles from the clients site, but must have missed these.  Will address and search for any others present.  Thanks!